prow/git/git.go

```diff
@@ -507,11 +507,21 @@ func (r *Repo) MergeCommitsExistBetween(target, head string) (bool, error) {
 }
 
 // ShowRef returns the commit for a commitlike. Unlike rev-parse it does not require a checkout.
-func (i *Repo) ShowRef(commitlike string) (string, error) {
-	i.logger.Infof("Getting the commit sha for commitlike %s", commitlike)
-	out, err := i.gitCommand("show-ref", "-s", commitlike).CombinedOutput()
+func (r *Repo) ShowRef(commitlike string) (string, error) {
+	r.logger.Infof("Getting the commit sha for commitlike %s", commitlike)
```

Comment:
consider using a field for commitlike

Reply:
this PR only renamed `i` to `r` for consistency reason, surprisingly github displays diff like this.

I can fix all the other logger in a separate PR